### DISCUSSION TITLE
Is there OSC Support?

### DISCUSSION THREAD

**Discussion by GoldenMass** [vrcx-team/VRCX#531] [Q&A]
I am currently making an OSC application that is like an advance utility program and was wondering if there was OSC. It will be useful if I would be able to access VRCX ingame without opening the steam desktop or clicking out of the window.

**Comment by Myrkie** [comment]
refer to https://github.com/vrcx-team/VRCX/issues/523

**Reply by GoldenMass** [reply]
I see thank you for answering.
